frontend/src/components/doctor-details/doctor-details.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { RootState } from 'common/types';
+import { useDispatch, useSelector } from 'react-redux';
+import { DoctorsActionCreator } from 'store/slices';
+import { DataStatus } from 'common/enums';
+import { useParams } from 'react-router-dom';
+import phoneIcon from 'assets/images/phone.svg';
+import checkIcon from 'assets/images/icons/check.svg';
+
+import styles from './styles.module.scss';
+
+interface ParamTypes {
+  id: string;
+}
+
+const DoctorDetails: React.FC = () => {
+
+  const { doctorDetails, dataStatus } = useSelector(({ doctors }: RootState) => ({
+    doctorDetails: doctors.doctorDetails,
+    dataStatus: doctors.dataStatus,
+  }));
+  const dispatch = useDispatch();
+  const { id } = useParams<ParamTypes>();
+
+  React.useEffect(() => {
+    dispatch(DoctorsActionCreator.getDoctorDetailsAsync(id))
+  }, [])
+
+  if (dataStatus === DataStatus.PENDING) {
+    return <div>...Loading</div>;
+  }
+
+  return (
+    <div className={styles.doctorsDetailsContainer}>
+      <div className={styles.personalDataContainer}>
+        <img className={styles.doctorImage} src={doctorDetails?.imagePath} width="58" height="60" loading="lazy"
```

Comment:
```suggestion
        <img className={styles.doctorImage} src={doctorDetails.imagePath} width="58" height="60" loading="lazy"
```
Remove all optional chaining